src/subcommand/giraffe_main.cpp

```diff
@@ -856,6 +865,18 @@ int main_giraffe(int argc, char** argv) {
                     cluster_coverage = score;
                 }
                 break;
+
+            case 'U':
+                {
+                    auto maxmin = parse<Range<size_t>>(optarg);
+                    if (maxmin <= 0) {
```

Comment:
I wonder if this will always quite work with the Range type, if someone specifies a range to values to try.

Maybe it's time to rip out the Range feature; we used it for like one grid search once and I don't think we use it anymore.